docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -199,8 +206,6 @@ data:
 EOF
 ----
 +
-That service removes pre-FFU workarounds and configures Compute
-services for Ceph storage backend.
 Provided above resources should contain a cell-specific configurations.
```

Comment:
What does “provided above resources” refer to? And I think we agreed to hide multi-cell references until FR1?

Reply:
yes, we should hide multicell specifics for FR1

Reply:
I suggest changing the sentence to:

The resources provided in the `ConfigMap` should contain cell-specific configurations.